prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -857,6 +879,12 @@ class ActionHistory {
                 it.writeStringField("timestamp", DateTimeFormatter.ISO_INSTANT.format(Instant.now()))
                 it.writeStringField("topic", report.schema.topic)
                 it.writeNumberField("reportItemCount", report.itemCount)
+                if (action.sendingOrg != null && action.sendingOrgClient != null)
```

Comment:
Showing 3 new fields in the response json:  sending org, http return status, external file name.